client/layout/guided-tours/main-tour.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+import {
+	makeTour,
+	Tour,
+	Step,
+	Next,
+	Quit,
+	Continue,
+	Link,
+} from 'layout/guided-tours/config-elements';
+import {
+	isNewUser,
+	selectedSiteIsPreviewable,
+	selectedSiteIsCustomizable,
+	previewIsNotShowing,
+	previewIsShowing,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const context = state =>
+	true || isNewUser( state );
+
+export const MainTour = makeTour(
+	<Tour name="main" version="20160601" path="/" context={ context }>
```

Comment:
I wanted to do something like `context={ and( isNewUser, isEnabled( 'guided-tours/main' ) ) }` here, but `tour=main` doesn't force the tour then.

I *think* forcing a tour via a qarg should respect the `path`, but override the `context` of the tour. This way tours can be accessed by links, but won't break by being shown on an incorrect path.